interface/themes/rtl_tabs_style_full.css

```diff
@@ -0,0 +1,468 @@
+/**
```

Comment:
Wouldn't it make much more sense to follow the strategy you used for the main theme to also provide rtl for the tab themes (note your group did that awhile back)?
Check out that mechanism and I would be happy to provide guidance/pointers as needed.

Reply:
Hi @bradymiller .
I preferred to continue with the new logic of the load assets that enables to add option for RTL languages in the config.yaml file and not with the logic of  'main theme' in the globals.php. I think it's cleaner and more simple.

Reply:
Agreed :)